runtime/src/main/java/org/corfudb/runtime/collections/StreamSubscriptionMetrics.java

```diff
@@ -39,6 +41,12 @@
                         .publishPercentiles(percentiles)
                         .publishPercentileHistogram(true)
                         .register(registry));
+        this.queueDurationTimer = MeterRegistryProvider.getInstance()
```

Comment:
Wondering if the MetricRegistryProvider can be obtained from the corfu runtime so all ufo metrics go to the same place?
but this is ok too.

Reply:
Pavel made this change recently, he can comment on this @PavelZaytsev